docs/2/technical/microshare-platform/app-config-examples/trending-air-quality-mcf88.md

```diff
@@ -17,47 +17,47 @@ The following is JSON example code for a Trending Air Quality MCF88 App.
   "navOptions": [
     {
       "title": "Realtime Desk Availability",
-      "link": "https://dapp.microshare.io/apps/view/5e0f8cd22b0000d1c2607d32"
+      "link": "https://dapp.microshare.io/apps/view/5e0f8cd22b0000d1c260????"
```

Comment:
these may be standard views, so the actual id may matter. consult with lisa

Reply:
I'm going to come back to this for the next pr. I talked with Lisa and there's a little bit of changing that needs to be done there.